cli.js

```
@@ -96,5 +97,8 @@ function handleFkillError(processes) {
 if (cli.input.length === 0) {
 	init(cli.flags);
 } else {
-	fkill(cli.input, cli.flags).catch(() => handleFkillError(cli.input));
+	Promise.all(cli.input.map(x => x[0] === ':' ? portToProc(parseInt(x.slice(1), 10)) : x))
+				.then(pids => pids.filter(pid => pid))
+				.then(pids => fkill(pids, cli.flags))
+				.catch(() => handleFkillError(cli.input));
```

This will swallow errors from `portToProc` which may be unintended here.